app/models/partnership.rb

```diff
@@ -4,6 +4,7 @@ class Partnership < ApplicationRecord
   belongs_to :school
   belongs_to :lead_provider
   belongs_to :cohort
+  belongs_to :delivery_partner
```

Comment:
Ok, now I am certain it duplicates info from `ProviderRelationship`. Uhh, the data model looks like a mess 😢 

A mess that could and should use a refactor soon.

Reply:
Thought about adding a reference to provider relationships to partnerships but that seemed more fraught with danger than what I did.